ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -496,6 +496,62 @@
 | L3 Gateway Inter-domain | True |
 {%         endif %}
 {%     endif %}
+{%     if router_bgp.address_family_vpn_ipv4 is arista.avd.defined %}
+
+### Router BGP VPN-IPv4 Address Family
+{%         if router_bgp.address_family_vpn_ipv4.route.import_match_failure_action is arista.avd.defined('discard') %}
+
+- VPN import pruning is __enabled__
+{%         endif %}
+{%         if router_bgp.address_family_vpn_ipv4.neighbors is arista.avd.defined %}
+
+#### VPN-IPv4 Neighbors
+
+| Neighbor | Activate |
```

Comment:
Since this PR is about adding route-map, I think you should also include route-map in the table here.

Reply:
Will make a PR with the following table format
```
| Neighbor | Activate | Route-map In | Route-map Out |
| -------- | -------- | ------------ | ------------- |
```